test/tasks/demo.test.js

```diff
@@ -39,27 +40,29 @@ describe('Demo task', function() {
 	});
 
 	describe('Build demos', function() {
-		it('should fail if there is not a bower.json file', function(done) {
-			demo(gulp)
-				.catch(function(err) {
+		it('should fail if there is not a bower.json file', function() {
+			return demo(gulp)
```

Comment:
mocha supports promises, this means that if the test fails it fails immediately rather than just timing out.